lib/Qi.pm

```diff
@@ -20,9 +26,17 @@ sub startup {
 
   # Normal route to controller
   $r->get('/')->to('example#welcome');
+  
+  my $api = $r->under('/api');
+  $api->get('/assets')->to('assets#get')->name('get');       # get assets
+  $api->post('/assets')->to('assets#insert')->name('test');     # insert asset
+  $api->put('/assets')->to('assets#update')->name('update');     # update asset
```

Comment:
this is the api structure I came up with on the fly. Definitely subject to change if/when we find a better way.